packages/components/pages/f-contact-preferences/src/services/providers/_tests/contactPreferences.api.test.js

```diff
@@ -0,0 +1,146 @@
+import ContactPreferencesApi from '../contactPreferences.api';
+
+let apiProvider;
+let httpMock;
+const httpGetSpy = jest.fn();
+const httpPostSpy = jest.fn();
+let cookiesMock;
+const cookiesSetSpy = jest.fn();
+const baseUrlMock = 'https://smartGatewayBaseUrl.com';
+const authTokenMock = 'some-auth-token';
+const conversationId = 'b7386108-95e6-4e73-9421-5b066c089153';
+const bodyMock = {
+    Preference: [{
+        DisplayName: 'Order status',
+        Sort: 1,
+        Key: 'orderstatus',
+        Push: false,
+        Email: true,
+        Sms: true
+    }, {
+        DisplayName: 'Review meal',
+        Sort: 2,
+        Key: 'reviewmeal',
+        Push: false,
+        Email: false,
+        Sms: false
+    }, {
+        DisplayName: 'News & offers',
+        Sort: 3,
+        Key: 'news',
+        Push: false,
+        Email: true,
+        Sms: false
+    }
+    ],
+    DeviceToken: null,
+    DeviceType: null,
+    PhoneNumber: null,
+    PreferenceVersionViewed: 0
+};
+
+describe('ContactPreferencesApi Provider', () => {
+    beforeEach(() => {
+        // Arrange mocks/spies
+        httpMock = {
+            get: httpGetSpy,
+            post: httpPostSpy
+        };
+        cookiesMock = {
+            set: cookiesSetSpy
+        };
+
+        // Arrange - sut
+        apiProvider = new ContactPreferencesApi({
+            httpClient: httpMock,
+            cookies: cookiesMock,
+            baseUrl: baseUrlMock
+        });
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('When creating a new instance', () => {
+        it('should not throw error when instance is created with valid parameters', () => {
+            // Act
+            const instance = () => new ContactPreferencesApi({
+                httpClient: httpMock,
+                cookies: cookiesMock,
+                baseUrl: baseUrlMock
+            });
+
+            // Assert
+            expect(instance).not.toThrowError();
+            expect(instance).toBeDefined();
```

Comment:
I suspect `expect(instance).toBeDefined()` might not be checking what you want it to check. It will probably always be defined because you have defined it as a function above.

```suggestion
            const createInstance = () => new ContactPreferencesApi({
                httpClient: httpMock,
                cookies: cookiesMock,
                baseUrl: baseUrlMock
            });

            // Assert
            let instance;
            expect(() => {
                instance = createInstance();
            }).not.toThrowError();
            expect(instance).toBeDefined();
```